cms/jinja2/templates/components/streamfield/image_block.html

```diff
@@ -1,4 +1,57 @@
-{% image(value.image, "width-1000", 'loading="lazy"' %}
-{% if value.caption %}
-    <p>{{ value.caption }}</p>
-{% endif %}
+<!--
+    What is considered a large vs small image can vary based on the context of use.
+-->
+{# templates/components/streamfield/image_block.html #}
+{% set large_img = image(value.image, "width-800") %}
+{% set small_img = image(value.image, "width-400") %}
+
+{% from "components/image/_macro.njk" import onsImage %}
+
+{# fmt:off #}
+<div>
+    {# Figure Title #}
+    {% if value.figure_title %}
+        <h3 class="ons-chart__title">{{ value.figure_title }}</h3>
+    {% endif %}
+
+    {# Figure Subtitle #}
+    {% if value.figure_subtitle %}
+        <p>{{ value.figure_subtitle }}</p>
+    {% endif %}
```

Comment:
Semantically, if we are using a `p` tag for the subtitle, we should wrap both elements in an `hgroup` ([reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/hgroup)). Also bear in mind that the `p` tag will most likely have some sort of styling applied to it used for paragraphs (e.g. margins).

Reply:
Done: [17bf248](https://github.com/ONSdigital/dis-wagtail/pull/437/commits/17bf248f741b3857a0cfe7ba741aaa00fb217067)